File: backend/models/NhaCungCap.js
const { DataTypes } = require("sequelize");
const sequelize = require("../config/db");

const NhaCungCap = sequelize.define(
  "NhaCungCap",
  {
    id: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
    ten_nha_cc: { type: DataTypes.STRING(255), allowNull: false },
    ma_so_thue: { type: DataTypes.STRING(50) },
    email: { type: DataTypes.STRING(255) },
    so_dien_thoai: { type: DataTypes.STRING(20) },
    dia_chi: { type: DataTypes.STRING(255) },
    quoc_gia: { type: DataTypes.STRING(100) },
    trang_thai: { type: DataTypes.STRING(20), defaultValue: "active" },
    ghi_chu: { type: DataTypes.TEXT },
  },
  {
    tableName: "NhaCungCap",
    timestamps: false,
  },
);

module.exports = NhaCungCap;

File: backend-csharp/Models/NhaCungCap.cs
using System;
using System.Collections.Generic;

namespace WebFashion.Api.Models;

public partial class NhaCungCap
{
    public long Id { get; set; }

    public string TenNhaCc { get; set; } = null!;

    public string? MaSoThue { get; set; }

    public string? Email { get; set; }

    public string? SoDienThoai { get; set; }

    public string? DiaChi { get; set; }

    public string? QuocGia { get; set; }

    public string TrangThai { get; set; } = null!;

    public string? GhiChu { get; set; }

    public virtual ICollection<PhieuNhapHang> PhieuNhapHangs { get; set; } = new List<PhieuNhapHang>();

    public virtual ICollection<SanPham> SanPhams { get; set; } = new List<SanPham>();
}
